README.md

```diff
@@ -1,10 +1,70 @@
 # disblob
+A stylesheet(css) generator for use with Discord.
+***
+***All original code is done by the insane but genius [diamondburned](https://github.com/diamondburned/)***
+> [Master repo](https://github.com/diamondburned/disblob)
 
-A stylesheet generator for use with Discord.
+***This fork is an attempt to refine the `README` and make it easier for others to setup!***
+***
+# Status
+|   |Status |	|
+|--|--|--|
+|   |Edits: |	|
+|  | `README` Only |  |
 
+|   |Fork|  |
+|--|--|--|
+|  | Fork Only |  |
+|  |  (PR [Open](https://github.com/diamondburned/disblob/pull/1))  |  |
+
+|  | Testing |  |
+|--|--|--|
+|  | 		-Client + Firefox-		 |  |
+|  |-  (Tests have been done with the client and Firefox)  |  
+***
+ - [x] Client (BD(BetterBiscord)~(100% Working!))
+ - [ ] Client (Other css injectors)
+ - [ ] Browsers (Chrome, etc. (Use [stylus](https://github.com/openstyles/stylus) to inject css)
```

Comment:
Chrome is tested by me.

Reply:
Perfect, testing on other clients isn't needed, it most likely makes no difference